src/transforms/projectionpursuit.jl

```diff
@@ -0,0 +1,165 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    ProjectionPursuit(tol=1e-6, maxiter=100, deg=5, n=100)
+
+The projection pursuit transform converts any multivariate distribution
+to a standard multivariate Gaussian. The tolerance `tol` and maximum
+number of iterations `maxiter` control the iterative process. Optionally,
+specify the degree `deg` of the Legendre polynomials and the number
+of samples `n` from the multivariate standard Gaussian in the stopping
+criteria.
+
+See [https://doi.org/10.2307/2289161](https://doi.org/10.2307/2289161)
+for further details.
+
+"""
+
+struct ProjectionPursuit{T} #<: Transform
+  tol::T
+  perc::T
+  maxiter::Int
+  deg::Int
+  n::Int
+end
+
+ProjectionPursuit(tol=1e-6, perc=.9, maxiter=100, deg=5, n=100) =
+  ProjectionPursuit{typeof(tol)}(tol, perc, maxiter, deg, n)
+
+using LinearAlgebra
+using Statistics
+using DataFrames
+using Distributions
+using Optim
+
+#Transforms a row of random variables into a convex combination 
+#random variable with values in [-1,1] and standard normal distribution.
+function rscore(table, α)
+  ᾱ = (1/norm(α)) .* α .* (norm(α) != 1) + α .* (norm(α) == 1)
+  X = Matrix(table) * ᾱ
```

Comment:
```suggestion
  X = Tables.matrix(table) * ᾱ
```
To convert a table in a matrix use `Tables.matrix(table)` function from Tables.jl package.